### PR TITLE
Change edit to rename

### PR DESCRIPTION
**Pull request by Mav-Ivan**
image

### CODE REVIEW

File: src/containers/my-resources/my-resources-table/MyResourcesTable.tsx [hunk 1]
```diff
@@ -88,7 +90,7 @@ const MyResourcesTable = <T extends TableItem>({
 
   const rowActions: TableRowAction[] = [
     {
-      label: t('common.edit'),
+      label: !rename ? t('common.edit') : t('common.rename'),
```
Comment (Tolik170) [inline src/containers/my-resources/my-resources-table/MyResourcesTable.tsx:93]: As rename is used only for Categories, you can check it by **ResourcesTabsEnum.Categories**

label: resource === ResourcesTabsEnum.Categories ? t('common.rename') : t('common.edit')

Comment (Mav-Ivan) [reply src/containers/my-resources/my-resources-table/MyResourcesTable.tsx]: yeah thats better

File: src/containers/my-resources/my-resources-table/MyResourcesTable.tsx [hunk 2]
```diff
@@ -30,6 +31,7 @@ interface MyResourcesTableInterface<T>
   }
   services: { deleteService: (id?: string) => Promise<AxiosResponse> }
   pagination: PaginationProps
+  rename?: boolean
```
Comment (Tolik170) [inline src/containers/my-resources/my-resources-table/MyResourcesTable.tsx:34]: so, you don`t need this prop anymore

**Review by Tolik170 — APPROVED** [review]

File: src/containers/my-resources/my-resources-table/MyResourcesTable.tsx (hunk 2 above)
Comment (Mav-Ivan) [reply src/containers/my-resources/my-resources-table/MyResourcesTable.tsx:34]: thanks

**Review by dmtrth25 — APPROVED** [review]

**Review by TSlashDreamy — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**